index.html

```diff
@@ -29,5 +31,115 @@ <h3> I'm a software developer </h3>
         </ul>
       </div>     
   </section>
+  <section class="works">
+    <div class="recent-works"> 
+      <h1> My recent works </h1>
+    </div>
+    <img src="img/Placeholder.svg" alt="">
+    <h2> Multi-Post Stories</h2>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+  <div class="box">
+    <button type="button"> CSS</button>
+    <button type="button">html</button>
+    <button type="button">Bootstrap</button>
+    <button type="button">Ruby</button>
```

Comment:
-  [ ] Please ensure to use list item tag ```<li>``` for the languages and they should be wrapped in an unordered list ```<ul>```

For example
```
<ul>
  <li>html</li>
</ul>
```